setup.py

```diff
@@ -258,6 +259,7 @@ def run(self):
 
 extras["tf"] = deps_list("tensorflow", "onnxconverter-common", "tf2onnx", "tensorflow-text", "keras-nlp")
 extras["tf-cpu"] = deps_list("tensorflow-cpu", "onnxconverter-common", "tf2onnx", "tensorflow-text", "keras-nlp")
+extras["pygtrie"] = deps_list("pygtrie")
 
 extras["torch"] = deps_list("torch", "accelerate")
```

Comment:
```suggestion
extras["torch"] = deps_list("torch", "accelerate", "pygtrie")
```

Try this instead :) The `extras["pygtrie"]` you defined is not installed, as it is not included in `extras["all"]`